### ISSUE TITLE
Convert EBM model to ONNX to get rid of interpret dependency

### ISSUE THREAD

**Issue by Hoeze** [gagneurlab/absplice#2]
pip-installing interpret pulls a myriad of other awful pip dependencies like e.g. seaborn.
=> https://github.com/interpretml/ebm2onnx

**Comment by paulbkoch** [comment]
Another option is to take a dependency on interpret-core.  interpret-core has no dependencies by default, and you can pull in whichever ones you need.  For example, if you only need to build ebms, you can use:

pip install interpret-core[required,ebm]

Other optional flags are:
https://interpret.ml/docs/installation-guide.html#install-interpret-with-every-dependency

The interpret package is meant to be a heavyweight full featured option which exposes other interpretability methods, and also has visualization.

**Comment by Hoeze** [comment]
Thanks, if restricting to `interpret-core==0.2.7` is sufficient, this is a very elegant solution!
CC @neverov-am

**Comment by paulbkoch** [comment]
Yes, this mechanism is available in 0.2.7 as well, and the flags are still the same.

The v0.2.7 setup.py requirements:
https://github.com/interpretml/interpret/blob/ba6ba620936d3c0d21d66418b6d8e3a2834e265a/python/interpret/setup.py#L52
